lib/widgets/add_todo_dialogue.dart

```diff
@@ -1,27 +0,0 @@
-import 'package:flutter/material.dart';
```

Comment:
Though the tests pass, it is better practice to separate your widgets. For eg the add Todo-dialogue, add it to add_todo_dialogue.dart file, same for todo_list_item.dart.